swift-lint-sample/AppDelegate.swift

```diff
@@ -23,9 +23,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UISplitViewControllerDele
         return true
     }
 
-    func applicationWillResignActive(application: UIApplication) {
-        // Sent when the application is about to move from active to inactive state. This can occur for certain types of temporary interruptions (such as an incoming phone call or SMS message) or when the user quits the application and it begins the transition to the background state.
-        // Use this method to pause ongoing tasks, disable timers, and throttle down OpenGL ES frame rates. Games should use this method to pause the game.
+    func applicationWillResignActive(application: UIApplication)  {
```

Comment:
ERROR: [Warning][opening_brace] Opening Brace Spacing
Opening braces should be preceded by a single space and on the same line as the declaration.